public/app/plugins/datasource/azuremonitor/components/LogsQueryEditor/LogsManagement.tsx

```diff
@@ -1,31 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import { InlineField, RadioButtonGroup } from '@grafana/ui';
+import { ConfirmModal, InlineField, RadioButtonGroup } from '@grafana/ui';
 
 import { AzureQueryEditorFieldProps } from '../../types';
 
 import { setBasicLogsQuery, setDashboardTime, setKustoQuery } from './setQueryValue';
 
 export function LogsManagement({ query, onQueryChange: onChange }: AzureQueryEditorFieldProps) {
+  const [basicLogsAckOpen, setBasicLogsAckOpen] = useState<boolean>(false);
   return (
     <>
+      <ConfirmModal
+        isOpen={basicLogsAckOpen}
+        title="Basic Logs Queries"
+        body="Are you sure you want to switch to Basic Logs?"
+        description="Basic Logs queries incur cost based on the amount of data scanned. Please acknowledge this panel is subject to per-query costs."
+        confirmText="I Acknowledge"
```

Comment:
```suggestion
        confirmText="Confirm"
```

Reply:
I think this might be the default text actually 🤔

Reply:
I tried removing it to see if there was a default, but the prop is required, so added your suggestion :)